src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -379,7 +379,20 @@ public function load(array $configs, ContainerBuilder $container): void
         $this->registerPropertyAccessConfiguration($config['property_access'], $container, $loader);
         $this->registerSecretsConfiguration($config['secrets'], $container, $loader, $config['secret'] ?? null);
 
-        $container->getDefinition('exception_listener')->replaceArgument(3, $config['exceptions']);
+        $exceptionListener = $container->getDefinition('exception_listener');
+
+        $loggers = [];
+        foreach ($config['exceptions'] as $exception) {
+            if (!isset($exception['log_channel'])) {
+                continue;
+            }
+            $loggers[$exception['log_channel']] = new Reference(sprintf('monolog.logger.%s', $exception['log_channel']));
```

Comment:
Does it work for applications that dont use monolog, but other logging library?

Reply:
No because I don't find a way to make the logger tag generic and compatible with other loggers. In fact, on the PS3, there is no specification for channel

Reply:
I suppose we want to fallback to the default logger when the defined channel doesn't exist:
```suggestion
            $loggers[$exception['log_channel']] = new Reference(\sprintf('monolog.logger.%s', $exception['log_channel']), ContainerInterface::NULL_ON_INVALID_REFERENCE);
```